app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -98,14 +95,118 @@ public function panel(Panel $panel): Panel
 
     public function boot(): void
     {
+        //If you change colors here also change in tailwind.config
         FilamentColor::register([
```

Comment:
These colors should be registered outside of the panel, for things like the frontend form builder. Can we move it to either AppServiceProvider, or a new FilamentServiceProvider?